src/types/structure.ts

```diff
@@ -94,13 +93,17 @@ export default class Structure {
 
     /**
      * Paste World chunk into this world
+     * @param {args} options Options can be inherited by children classes who have more control over block placement control flow
      */
-    public paste(xt: number, yt: number, data: Structure) {
+    public paste(xt: number, yt: number, data: Structure, options?: never): Promise<any> {
```

Comment:
Method is not `async` so a promise is not necessary

Reply:
World inherits from the method